doc/pages/user/050-query_device.md

```diff
@@ -76,6 +79,13 @@ _Sample Response_
 
 This returns the Interfaces which the device reported in its Introspection *and* which are known to the Realm.
 
+Arbitrary information can be added to the device by means of `metadata`: they allow to store any number of string values associated to a corresponding string key.
+To set, modify and delete `metadata`, a `PATCH` on the device endpoint is required:
+```
+PATCH appengine.api.<your astarte domain>/v1/test/devices/f0VMRgIBAQAAAAAAAAAAAA
```

Comment:
The URL is in the "old" format. Should be `api.<your astarte domain>/appengine/v1`...